src/editor/editorinteractive.h

```diff
@@ -260,10 +263,16 @@ class EditorInteractive : public InteractiveBase {
 	std::unique_ptr<Tools> tools_;
 	std::unique_ptr<EditorHistory> history_;
 
-	bool draw_resources_ = true;
-	bool draw_immovables_ = true;
-	bool draw_bobs_ = true;
-	bool draw_grid_ = true;
+        enum DisplayFlags : uint8_t {
+                kNone = 0,
```

Comment:
I think you can fix clang builds if you make this an `enum class`. Then all uses will need `DisplayFlags::`, but that's actually better anyway.

Reply:
I don't think the bitwise operators work with enum class

Reply:
(Disclaimer: This is the point where this review should be taken over by someone more proficient in C++ than me.)

Ah, OK, I see it's private to the class EditorInteractive, so not that bad after all. Still, how about making all the names start with `kDisplay` then for clarity?

Reply:
Yes, thanks! I actually realized that I had misunderstood what #4335 was about and changed it completely. Not sure if the editor-specific flags should go to editorinteractive.h or interactive_base.h, though.